lib/http_requests.py

```diff
@@ -74,28 +74,84 @@ def get_http_data(url):
     return yml_data
 
 
-def get_advisories(branch_name):
-    url = f"{os.environ['GITHUB_RAW_CONTENT_URL']}/{branch_name}/releases.yml"
+def get_particular_advisory(data):
+    """
+    Function to get the advisories from a particular z-stream release
+    :data: The data of a particular version
+    """
+    try:
+        advisories = data['assembly']['group']['advisories']
+        if -1 in advisories.values() or 1 in advisories.values():
+            return None
+    except Exception:
+        return None
 
+    return advisories
+
+
+def get_brew_event_id(data):
+    """
+    Function to get the brew_event id from the releases.yml file
+    :data: The yml for a particular z-stream release
+    """
     try:
-        yml_data = get_http_data(url)['releases']
+        brew_event_id = data['assembly']['basis']['brew_event']
+        return brew_event_id
+    except KeyError:
+        return None
 
-        advisory_data = []
-        for version in yml_data:
-            try:
-                advisories = yml_data[version]['assembly']['group']['advisories']
-                brew_event = yml_data[version]['assembly']['basis']['brew_event']
-            except:
-                continue
 
-            if -1 in advisories.values() or 1 in advisories.values():
-                continue
-            advisory_data.append([brew_event, advisories])
+def get_advisories(branch_name):
```

Comment:
this method name wasn't very good before, but now it is really not obvious what this is doing. can you add a comment about what it's supposed to return, and maybe a little below about how it's supposed to work? i have a feeling this can be simplified but i'm having a hard time following.

test cases might also make this more obvious.

Reply:
Added comments. It was implemented to get all the advisories from` group.yml` before before changed it to `releases.yml `and added the inheritance feature. We only care about the latest 2 advisories, but I kept the function as is so that it returns everything in the releases.yml file, if in case art-dash-ui needs in the future.